### PR TITLE
Nazanin/career updates

### PR DESCRIPTION
**Pull request by nazaninreihani**

### CODE REVIEW

File: src/templates/static/about/job_descriptions.jsx [hunk 1]
```diff
@@ -56,95 +56,96 @@ const JobDescriptions = () => (
...
+                    { header: it.L('Front-End Developer'),  subsection: 'frontend_developer',   text: it.L('As our Front-End Developer, you will be taking on the challenge of developing and maintaining advanced applications and interfaces that connect clients with our patented trading system.') },
+                    { header: it.L('Perl Developer'),       subsection: 'perl_developer',       text: it.L('As our Perl Developer, you will be taking on the challenge of writing robust, high-quality, and production-ready code. Your work will greatly contribute to the architecture that drives our high-traffic binary options trading website.'), className: 'one-line nowrap' },
+
+                    { header: it.L('Cryptocurrency/Blockchain Developer'), subsection: 'cryptocurrency_Blockchain_developer',     text: it.L('As our Cryptocurrency/Blockchain Developer, you will be taking on the challenge of revamping our existing cashier system and infrastructure for our ICO launch. You will also drive all our future blockchain-based projects. Your work will greatly contribute to the architecture that drives our high-traffic binary options trading website.'), className: 'one-line nowrap' },
```
Comment (negarn) [inline src/templates/static/about/job_descriptions.jsx]: no need for the extra space before `text:`

File: src/templates/static/about/job_details.jsx [hunk 2]
```diff
@@ -106,473 +109,642 @@ const JobDetails = () => (
...
 
                             <UlText
-                                text={it.L('Requirements:')}
+                                text={it.L('Responsibilities:')}
                                 className='bullet'
                                 items={[
-                                    { text: it.L('A strong and passionate interest in Social Media, Marketing, and Advertising') },
-                                    { text: it.L('The ability to communicate effectively in English, in both oral and written form') },
-                                    { text: it.L('A university degree in Marketing, Communication, Business Administration, or related discipline') },
+                                    { text: it.L('Test the Binary.com website, applications, backend, and back-office systems.') },
```
Comment (negarn) [inline src/templates/static/about/job_details.jsx]: please use a placeholder for `Binary.com` to use `it.website_name` instead

File: src/templates/static/about/job_details.jsx [hunk 3]
```diff
@@ -106,473 +109,642 @@ const JobDetails = () => (
...
+                        <div className='invisible' id='affiliate_country_manager'>
+                            <h1>{it.L('Affiliate/Country Manager')}</h1>
+
+                            <p>{it.L('The Country Manager is expected to acquire, service, manage, and expand our network of active affiliates in Russian-speaking countries. You will also contribute your energy, skills, and knowledge of the local business culture to increase our rapid growth rate worldwide.')}</p>
+                            <p>{it.L('You are also expected to help us promote our cutting-edge trading platform that has been one of the most recognised in the binary options trading industry for over 18 years.')}</p>
 
                             <UlText
-                                text={it.L('Requirements:')}
+                                text={it.L('Responsibilities:')}
                                 className='bullet'
```
Comment (negarn) [inline src/templates/static/about/job_details.jsx]: please use a placeholder for `<a href=\'https://www.binary.com\' target=\'_blank\'>www.binary.com</a>` so we don't send this to translation

File: src/templates/static/about/job_details.jsx [hunk 4]
```diff
@@ -106,473 +109,642 @@ const JobDetails = () => (
...
 
-                            <p>{it.L('[_1] seeks a Project Coordinator to help drive the production of promotional content and rollout of marketing campaigns, designed to engage new prospects and current customers based throughout the globe.', it.website_name)}</p>
-                            <p>{it.L('Join our team of bright, talented professionals. Work collaboratively with us to enhance and promote a high-tech platform that has been one of the most recognised in the binary options trading industry for over 15 years.')}</p>
+                            <p>{it.L('As a strong proponent of open source, we encourage publication of findings, methods, and tools via GitHub and our technical blog at https://tech.binary.com/ You will also assist our developers in understanding and patching the bugs that you find.')}</p>
```
Comment (negarn) [inline src/templates/static/about/job_details.jsx]: I assume that this is supposed to be a link? `https://tech.binary.com/`, please use a placeholder too

File: src/templates/static/about/job_descriptions.jsx [hunk 5]
```diff
@@ -56,95 +56,96 @@ const JobDescriptions = () => (
 
             <Row
                 id='information_technology'
-                title={it.L('Information Technology')}
+                title={it.L('IT')}
```
Comment (kellybinary) [inline src/templates/static/about/job_descriptions.jsx]: i think it's better to keep it as `Information Technology`

File: src/templates/static/about/job_details.jsx [hunk 6]
```diff
@@ -724,7 +724,7 @@ const JobDetails = () => (
                         <div className='invisible' id='copywriter'>
                             <h1>{it.L('Copywriter')}</h1>
 
-                            <p>{it.L('[_1] Marketing team oversees all our marketing and advertising initiatives, such as our affiliate programme, social media presence, webinars, and emails. As our Copywriter, you will help us to craft copy and content that engages our audiences –– both online and offline.', it.website_name)}</p>
+                            <p>{it.L('[_1]\'s Marketing team oversees all our marketing and advertising initiatives, such as our affiliate programme, social media presence, webinars, and emails. As our Copywriter, you will help us to craft copy and content that engages our audiences –– both online and offline.', it.website_name)}</p>
```
Comment (negarn) [inline src/templates/static/about/job_details.jsx]: can you replace any instances of `––` in this PR with the shorted `-`? `––` doesn't render well in browser

Comment (nazaninreihani) [reply src/templates/static/about/job_details.jsx]: Done :)

**Pull request merged**